packages/create-flex-plugin/README.md

```diff
@@ -32,7 +32,21 @@ npm install -g create-flex-plugin
 create-flex-plugin plugin-demo
 ```
 
-We do not support `yarn` at the moment.
+#### Using `yarn`:
+
+```bash
+# Via npm init
+npm init flex-plugin plugin-demo --yarn
+
+# Via npx
+npx create-flex-plugin plugin-demo --yarn
+
+# Install globally
+npm install -g create-flex-plugin
+create-flex-plugin plugin-demo --yarn
+```
+
+_Note_: Support for `yarn` only exists for `@twilio/flex-ui > 1.18.0`. If you are upgrading from an older version of the plugin-builder, you'll need to add add `react-emotion` and pin it to `9.2.6`.
```

Comment:
remove extra `add`

Reply:
This is a rebase with master; we need to update master and update this branch later.